proto/usr/lib/systemd/system/triton-install-lxd.service

```diff
@@ -0,0 +1,30 @@
+#
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+#
+
+#
+# Copyright 2020 Joyent, Inc.
+#
+
+#
+# This file is delivered as part of the Triton Platform Image.
+#
+# This file is in charge of calling the install-lxd script, which will
+# proceed with the required LXD configuration into the CN
+#
+
+[Unit]
+Description=Triton install-lxd service
+Documentation=https://linuxcontainers.org/lxd/docs/master/index
+After=network.target zfs.target triton-fs.service
```

Comment:
I don't see a triton-fs.service anywhere in this wad. Does this need correcting? Perhaps triton-hostid.service?

Reply:
Ah, that needs to be removed.

It's something I did at one point to try to get factory reset and noimport working, but later realized that all of that is managed through the `live-bottom/syspool` script.

Reply:
fixed now.